migrations/20210401200001-migrate-virtual-cards-from-payout-methods.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+    await queryInterface.addColumn('Expenses', 'VirtualCardId', {
+      type: DataTypes.STRING,
+      references: { key: 'id', model: 'VirtualCards' },
+      onDelete: 'SET NULL',
+      onUpdate: 'CASCADE',
+      allowNull: true,
+    });
+
+    await queryInterface.addColumn('ExpenseHistories', 'VirtualCardId', {
+      type: DataTypes.STRING,
+      references: { key: 'id', model: 'VirtualCards' },
+      onDelete: 'SET NULL',
+      onUpdate: 'CASCADE',
+      allowNull: true,
+    });
+
+    await queryInterface.sequelize.query(`
+      INSERT INTO "VirtualCards" (
+        "id", "HostCollectiveId", "CollectiveId", "last4", "data", "createdAt", "updatedAt"
+      ) SELECT
+        pm."data"->>'token' AS "id",
+        c."HostCollectiveId" AS "HostCollectiveId",
+        pm."CollectiveId" AS "CollectiveId",
+        pm."name" as "last4",
+        pm."data" as "data",
+        pm."createdAt" as "createdAt",
+        pm."updatedAt" as "updatedAt"
+      FROM
+        "PayoutMethods" pm
+      INNER JOIN "Collectives" c
+        ON pm."CollectiveId" = c.id
+      WHERE
+        pm."deletedAt" IS NULL
+        AND pm."type" = 'CREDIT_CARD';
+    `);
+
+    await queryInterface.sequelize.query(`
+      UPDATE "Expenses"
+        SET "VirtualCardId" = "PayoutMethods"."data"->>'token'
```

Comment:
Shouldn't we also set `PayoutMethodId = NULL`?

Reply:
Definitely!